.changeset/lovely-worms-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
```

Comment:
Don’t know if this needs a major bump or not. If it does, we’ll need to wait a while before merging this. @matthewp & @bluwy are probably best place to confirm.

Reply:
> Don’t know if this needs a major bump or not. If it does, we’ll need to wait a while before merging this. @matthewp & @bluwy are probably best place to confirm.

My bad,  I will chang into patch

Reply:
I think the changes look good, but this indeed seems like a major bump as it would invalidate all Astro projects cache, and affect integrations that relied on the old default paths (e.g. like in this PR the image, deno, netlify integrations).

We should probably do this for Astro 2.0, and also add changesets major for the image, deno, netlify integrations

Reply:
Ok, I will change it to major late because I have had a positive recently